lua/autorun/client/cl_contact_forms.lua

```diff
@@ -238,6 +239,32 @@ local function makePlayerDropdownField( question, parent )
     return ComboBox
 end
 
+local function makeAllPlayerDropdownField( question, parent )
+    local plys = {}
+    for _, ply in pairs( player.GetAll() ) do
+        if ply ~= LocalPlayer() then table.insert( plys, ply ) end
+    end
+
+    return makePlayerDropdownField( question, parent, plys )
+end
+
+local function makeStaffDropDownField( question, parent )
+    local plys = {}
+
+    local isStaff = {
+        ["sentinel"] = true,
+        ["moderator"] = true,
+        ["admin"] = true,
+        ["owner"] = true
```

Comment:
I'd recommend creating a `local StaffGroups` outside of this function, local to this file.
Then you aren't re-creating the table every time